packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json

```diff
@@ -19,7 +19,7 @@
   "billing_commitment_payment_method_none": "You do not have a valid payment method.",
   "billing_commitment_payment_method_add": "Add a new payment method ",
   "billing_commitment_summary": "Finalizing",
-  "billing_commitment_summary_description_periodic": "Starting {{ renewDate }} your default payment method will automatically be charged [$XX.XX] every month for {{ serviceName }} until {{ engagementEndDate }}.",
+  "billing_commitment_summary_description_periodic": "Starting {{renewDate}} your default payment method will automatically be charged {{price}} every month for {{serviceName}} until {{engagementEndDate}}.",
```

Comment:
Missing a translate-value, isn't it?

see: https://github.com/ovh/manager/blob/fix/commitment-content-english/packages/manager/modules/billing-components/src/components/commitment/commitment.html#L151-L153

Reply:
I didn't pay attention to the actual diff, sorry.

https://github.com/ovh/manager/pull/5520/files#diff-a8173cdfcadae3ac58bfba01e641ac0d0b004c225e753909ca9e69ae4946d490R145